templates/asset/ratings/templates.yaml

```diff
@@ -0,0 +1,24 @@
+dataset: asset
+subset: ratings
+templates:
+  09b2a13b-cba6-4473-8a46-3fa24be71ce2: !Template
+    id: 09b2a13b-cba6-4473-8a46-3fa24be71ce2
+    jinja: "{% set questions= [ \"Does the second sentence better convey the information?\"\
+      ,  \"Is the second sentence more fluent?\", \"Is the second sentence easier\
+      \ to understand?\"] %}\n\nFirst sentence: {{original}}\n\nSecond sentence: {{simplification}}\n\
+      \n{{questions[aspect]}} \n\n|||\n\n{% if rating > 50 %}\n    Yes\n{% else %}\n\
+      \    No\n{% endif %}"
+    name: asset_ratings1
+    reference: Taking questions from the original paper, we use rating to establish
+      a binary classification problem.
+    task_template: false
+  d2bed959-29ab-4962-a106-dc91c00f3f03: !Template
+    id: d2bed959-29ab-4962-a106-dc91c00f3f03
+    jinja: "{% set statements= [ \"the second sentence expresses the underlying meaning\
+      \ the best.\",  \"the second sentence if more fluent.\", \"the second sentence\
+      \ is easier to read and understand.\"] %}\n\nFirst sentence: {{original}}\n\n\
+      Second sentence: {{simplification}}\n\nRate the following statement from 0 (strongly\
+      \ disagree) to 100 (strongly agree): {{statements[aspect]}} \n\n|||\n\n{{rating}}"
+    name: asset_ratings2
+    reference: Require the model to output the rating
+    task_template: false
```

Comment:
this template looks like the original task to me right? if so, it should be `true`